### PR TITLE
Fix: RPG and Jarmen are not playing explosion effect (visuals)

### PR DESCRIPTION
**Pull request by commy2**
black magic

### CODE REVIEW

File: Patch104pZH/Design/Changes/v1.0/101_demo_infantry_death_voice.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ labels:
 
 links:
   - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/101
+  - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/2307
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/101_demo_infantry_death_voice.yaml]: This looks not accurate. This change fixes missing suicide death effects of Demo RPG Trooper and Jarmen Kell.

Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/101_demo_infantry_death_voice.yaml]: You're right. I just tested 1.04, and the bug exists there as well (not really surprising, we don't break stuff ;)

So this PR should not be marked "Patch104p" and get its own yaml.

**Comment by commy2** [comment]
Should also check Stinger Soldier.

**Comment by commy2** [comment]
We need to check in general what happens if you blow them up with EXPLODED weapons. Looks like no effects present.

**Review by xezon — APPROVED** [review]
very good tricks

**Pull request merged**